cmd/buildah/containers.go

```diff
@@ -261,12 +261,24 @@ func containerOutputUsingTemplate(format string, params containerOutputParams) e
 
 func containerOutputUsingFormatString(truncate bool, params containerOutputParams) {
 	if truncate {
-		fmt.Printf("%-12.12s  %-8s %-12.12s %-32s %s\n", params.ContainerID, params.Builder, params.ImageID, params.ImageName, params.ContainerName)
+		fmt.Printf("%-12.12s  %-8s %-12.12s %-32s %s\n", params.ContainerID, params.Builder, params.ImageID, truncateString(params.ImageName, 32), params.ContainerName)
 	} else {
 		fmt.Printf("%-64s %-8s %-64s %-32s %s\n", params.ContainerID, params.Builder, params.ImageID, params.ImageName, params.ContainerName)
 	}
 }
 
+func truncateString(str string, to int) string {
+	newStr := str
+	if len(str) > to {
+		const tr = "..."
+		if to > len(tr) {
+			to -= len(tr)
+		}
+		newStr = str[0:to] + tr
+	}
+	return newStr
+}
+
```

Comment:
@saschagrunert you're too fast for me with these commits!  I'm thinking that this function would probably better live in util/util.go.  It could be used elsewhere at some point in time.  @nalind any disagreement?

Reply:
No objection.

Reply:
Okay, I move it over to util/util.go